content/tooling/wrangler/sites.md

```diff
@@ -126,3 +127,7 @@ Wrangler will always ignore:
 ##### More about include/exclude patterns
 
 You can learn more about the standard patterns used for include and exclude in the [gitignore documentation](https://git-scm.com/docs/gitignore).
+
+### Customizing your Build
+
+Workers Sites projects use webpack by default. You can [bring your own webpack config](/tooling/wrangler/webpack/#bring-your-own-configuration), however it is important to be cognizant of your `entry` and `context` settings.
```

Comment:
Since this is for Workers Sites, maybe it would make sense to update this link to directly go to the "Using with Workers Sites" section you added?